orttraining/orttraining/core/framework/ortmodule_graph_builder.cc

```diff
@@ -182,6 +182,10 @@ Status OrtModuleGraphBuilder::OptimizeForwardGraph(const TrainingGraphTransforme
         graph_transformation_mgr.ApplyTransformers(forward_graph, static_cast<TransformerLevel>(i), *logger_));
   }
 
+  if (!config.optimized_pre_grad_filepath.empty()) {
```

Comment:
We probably no longer need the function 

`std::string GetForwardModel() const;`. We can remove it I think.

Reply:
Looks inference manger still uses it to get the model to save. So I would suggest to keep it now.